auctions/models.py

```diff
@@ -92,8 +99,21 @@ def __unicode__(self):
         return u'Issue for %s (%s)' % (self.url, self.state)
 
 
+@receiver(post_save, sender=Issue)
+def lookup_title(sender, instance, **kwargs):
+    try:
+        r = requests.get(instance.url)
+        title_search = re.search('(?:<title.*>)(.*)(?:<\/title>)', r.text)
+        if title_search:
+            title = HTMLParser.HTMLParser().unescape(title_search.group(1))
+            Issue.objects.filter(id=instance.id).update(title=title)
+    except:
+        pass
+
+
 class Claim(models.Model):
     STATUS_CHOICES = (
+        ('Submitted', 'Submitted'),
```

Comment:
Submitted is for a claim without votes.  Pending means some votes but not all.  Not sure if this is an important distinction.